### ISSUE TITLE
"Migrating a test from XCTest" does not mention parallel-by-default

### ISSUE THREAD

**Issue by MahdiBM** [swiftlang/swift-testing#863]
"Migrating a test from XCTest" does not mention parallel-by-default, which catches users off-guard when migrating from XCTest to swift-testing.

There should also be a sentence or two explaining what to do to go back to the old behavior of non-parallel tests.

I was in a meeting last night and some folks mentioned this issue.
Apparently Fastlane also does not support having the tests run in parallel mode? so they had to figure out the `--no-parallel` CLI command and use that to run the tests sequentially.

**Comment by grynspan** [comment]
We do mention it in [Parallelization.md](https://github.com/swiftlang/swift-testing/blob/main/Sources/Testing/Testing.docc/Parallelization.md), but a nod here wouldn't be a bad idea.

**Comment by iamleeg** [comment]
Is there anything left to do here now that #960 is merged?

**Comment by stmontgomery** [comment]
> Is there anything left to do here now that [#960](https://github.com/swiftlang/swift-testing/pull/960) is merged?

I don't think so! This issue appears to have been addressed well by #960. Looks like we just forgot to close this issue afterward.

**Comment by stmontgomery** [comment]
Fixed by #960
